scripts/install_prerequisites.sh

```diff
@@ -54,3 +55,11 @@ sudo apt-get install libxml2-dev
 sudo apt-get install libxmlsec1-dev
 sudo apt-get install libffi-dev
 sudo apt-get install liblzma-dev
+
+# Check if the Python 2 is available and if so install it. This is needed
+# because the dev_appserver requires Python 2 to work.
```

Comment:
This comment should have the link, too (otherwise it reads as an assertion without proof). Devs may not realize that the link is in the other file.